.github/workflows/main.yaml

```diff
@@ -91,6 +91,7 @@ jobs:
         shell: Rscript {0}
       - name: Style and render documents
```

Comment:
I think this needs to be a different workflow rather than a step of this job. This workflow only runs on conditions defined here: 

https://github.com/mlverse/torch/blob/e06a8843d2258aaf2649ab0b6a3a08976ab3e3af/.github/workflows/main.yaml#L2-L8

And we want to run now on PR comments which I think doesnt not trigger this workflow to run.

Reply:
Ok. I tested this locally and it does work on push, but I agree that other triggers would require a separate workflow. We could use the example style workflow you shared earlier and add our command lines.